dependabotAlerts.js

```diff
@@ -0,0 +1,120 @@
+"use strict";
+const _ = require("lodash");
+
+const Promise = require("bluebird");
+const token = process.env.GITHUB_TOKEN;
+const { Octokit } = require("@octokit/rest");
+const { sum } = require("lodash");
+const octokit = new Octokit({
+  auth: token,
+  // Set GitHub Auth Token in environment variable
+});
+
+async function getAlerts(repos) {
+  const sortedAlerts = [];
+  const blocks = [];
+  const summary = {};
+  return await Promise.map(repos, async ({ org, name }) => {
+    const alerts = await getVulnerabilities(org, name);
+    const criticalAlerts = _.filter(alerts, {
+      severity: "critical",
+      dismissed: false,
+    });
+    const highAlerts = _.filter(alerts, {
+      severity: "high",
+      dismissed: false,
+    });
+    const mediumAlerts = _.filter(alerts, {
+      severity: "moderate",
+      dismissed: false,
+    });
+    // Dependabot calls these "moderate", but SparkPost categorizes these as "medium"
+    mediumAlerts.forEach((mediumAlert) => (mediumAlert.severity = "medium"));
+
+    if (criticalAlerts.length > 0) {
+      blocks.push(await buildBlocks(criticalAlerts));
```

Comment:
as noted above, I don't believe there's anything asynchronous about `buildBlocks`, so you shouldn't need `await`